pkg/util/mount/nsenter_mount.go

```diff
@@ -282,7 +274,15 @@ func (mounter *NsenterMounter) MakeFile(pathname string) error {
 }
 
 func (mounter *NsenterMounter) ExistsPath(pathname string) (bool, error) {
-	_, err := mounter.Lstat(pathname)
+	// Evaluate symlinks on the host, but allow the path not to exist -
+	// evalHostSymlinks would return undistinguishable error when path did not
+	// exist and mustExist was true.
+	evaluatedPath, err := mounter.evalHostSymlinks(pathname, false /* mustExist */)
+	if err != nil {
+		return false, err
+	}
+	kubeletPath := mounter.getKubeletPath(evaluatedPath)
+	_, err = os.Lstat(kubeletPath)
```

Comment:
what was wrong with mounter.Lstat()?

Reply:
Reworked `mounter.Lstat()` to return `NotExist` when the target does not exist (it returned some indistinguishable error before).